dnf-behave-tests/features/config-best.feature

```diff
@@ -49,9 +49,15 @@ Scenario: When installing with option --nobest, install a package from repo with
 
 
 @bz1670776 @bz1671683
-Scenario: When installing with best=1 (default), fail on broken packages, and advise to use --nobest
+Scenario: When installing with best=1 set in dnf.conf, fail on broken packages, and advise to use --nobest
   Given I use the repository "dnf-ci-fedora"
-  Given I use the repository "dnf-ci-fedora-updates"
+    And I use the repository "dnf-ci-fedora-updates"
+    And I do not set config file
+    And I create file "/etc/dnf/dnf.conf" with
```

Comment:
What about to replace the conf by:
```
gpgcheck=1
installonly_limit=3
clean_requirements_on_remove=True
best=True
```
It is identical to our conf that we ship.

Reply:
I can do that, but other options are irrelevant for the test.

Reply:
I'd suggest that we (eventually, not necessarily in this PR) create the config file from the steps in the default location (`/etc/dnf/dnf.conf`). Then we can add a step like this:
```
Given I set in config file
      | key  | value |
      | best | True  |
```

Which would use the defaults and add this to it.

If we go that route, it seems preferable to add the full config to these scenarios now, as @j-mracek suggests, because then whoever implements what I just described will not have to think if the other settings in the default config have any impact on the scenarios.

Reply:
Yes, I like it. This has advantages of both - only relevant settings in the feature and still full config used for execution. The only problem is what the defaults are? The Fedora ones? We are about to start delivering different dnf.conf files for different distributions.

Reply:
> The only problem is what the defaults are? The Fedora ones? We are about to start delivering different dnf.conf files for different distributions.

Right now the defaults are the contents of `dnf-behave-tests/features/dnf.conf`, so in context of the suggested change we'd go with that and effectively make no changes to the configuration... :slightly_smiling_face: 

As to what put in the defaults, not sure, we might need to discuss that? But with generating the config in runtime we have much more control and can more easily test multiple configuration combinations. The current defaults are options that don't change per distribution...